workshops/views.py

```diff
@@ -817,23 +817,27 @@ def search(request):
                 sites = Site.objects.filter(
                     Q(domain__contains=term) |
                     Q(fullname__contains=term) |
-                    Q(notes__contains=term))
+                    Q(notes__contains=term)) \
+                    .order_by('fullname')
```

Comment:
Does ordering by fullname involve a function call per sorted element?  If so, what's the speed hit?